### DISCUSSION TITLE
Intermittent Ctrl-Click (open URL) failure

### DISCUSSION THREAD

**Discussion by andyl** [ghostty-org/ghostty#4219] [Q&A] [answered]
I run ghostty on two Linux desktops (both Xubuntu 22.04).  On SystemA, CtrlClick works perfectly to open the URL under the cursor in the browser.  On SystemB, the operation works sometimes, but fails approx 75% of the time.  After failure, if I keep CtrlClicking, it works after 2 or 6 tries.

Incidentally: I see the same intermittent failure when doing CtrlShiftClick in Neovim (see discussion #4203)  

What I notice:
- running "xdg-open <url>" works perfectly 100% of the time in both systems
- when I hover over the URL, then press Ctrl, the URL is underlined, and Ghostty displays the url in a little popover box in the lower-left 100% of the time.  when I release the Ctrl key, the popover always goes away.  But when I CtrlClick, and it fails, the popover just stays until the next Ctrl keypress.

Questions:
- does anyone else notice this intermittent failure?
- is this a bug?
- could this be fixed somehow with Ghostty config?

**Comment by andyl** [comment, marked as answer]
This seems to be working now with today's release of Ghostty 1.0.1.

**Reply by JakeWharton** [reply]
Probably fixed by https://github.com/ghostty-org/ghostty/pull/3903
